tests/numeric_comparison_tests.py

```diff
@@ -58,3 +58,17 @@ def less_than_or_equal_to_matches_when_actual_is_less_than_or_equal_to_value():
 def less_than_or_equal_to_description_describes_value():
     matcher = less_than_or_equal_to(42)
     assert_equal("less than or equal to 42", matcher.describe())
+
+
+@istest
+def close_to_matches_when_actual_is_close_to_value_plus_delta():
+    matcher = close_to(42, 1)
+    assert_equal(matched(), matcher.match(43))
+    assert_equal(matched(), matcher.match(41))
+    assert_equal(unmatched("was 40"), matcher.match(40))
```

Comment:
Is it worth including a distance in the error message, or is it just adding noise? e.g. `was 40 (2 away from 42)`. I'm not sure that's exactly the right format.

Reply:
Hamcrest have 
    
    <40> differed by <2>

Experimenting with 

    was 40 (differs from 42 by more than 1)

but it is noisier, and not very pleasant for the datetimes